packages/tupaia-web/src/features/Dashboard/ExportDashboard/ExportConfig.tsx

```diff
@@ -0,0 +1,178 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { useParams } from 'react-router';
+import downloadJs from 'downloadjs';
+import { Button, LoadingContainer } from '@tupaia/ui-components';
+import { DashboardItemTypes } from '@tupaia/types';
+import { useEntity, useProject } from '../../../api/queries';
+import { useExportDashboard } from '../../../api/mutations';
+import { MOBILE_BREAKPOINT } from '../../../constants';
+import { DisplayOptionsSettings, useExportSettings } from '../../ExportSettings';
+import { useDashboard } from '../utils';
+import { ExportSubtitle } from './ExportSubtitle';
+import { MailingListSection } from './MailingListSection';
+import { Preview } from './Preview';
+
+const ButtonGroup = styled.div`
+  padding-top: 2.5rem;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  width 100%;
+  align-items: start;
+  section + section {
+    margin-top: 1.5rem;
+    border-top: 0.1rem solid ${({ theme }) => theme.palette.text.secondary};
+    padding-top: 1.5rem;
+  }
+`;
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  flex-basis: 83.3333%;
+
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    flex-direction: row;
+  }
+`;
+
+const ExportSettingsContainer = styled.div`
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 2rem;
+
+  @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
+    width: 62%;
+    margin-right: 2rem;
+    margin-bottom: 0;
+  }
+`;
+
+const ExportSetting = styled.div`
+  border: ${({ theme }) => theme.palette.text.secondary};
+  border-width: 0.1rem;
+  border-style: solid;
+  border-radius: 7px;
+
+  .MuiFormGroup-root {
+    align-content: start;
+  }
+
+  .MuiFormControlLabel-label {
+    font-size: 0.875rem;
+  }
+
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+  padding: 1rem;
+
+  fieldset {
+    border: 0;
+  }
```

Comment:
Is there somewhere sensible we can apply this reset globally? In DataTark I’ve also just repeated it each time I’ve used `fieldset`, but it feels slightly clumsy. (Not sure if `themes.ts` is the right place for it)

Whether it’s here or somewhere global, [`<fieldset>` has a couple other bizarre defaults]((https://html.spec.whatwg.org/multipage/rendering.html#the-fieldset-and-legend-elements)) that we probably don’t want:

```suggestion
  fieldset {
    border: 0;
    margin: 0;
    min-width: 0;
    padding: 0;
  }
```

Reply:
Yes, I agree. I went with [this approach](https://v4.mui.com/customization/globals/#global-css)